pkg/config/constants.go

```diff
@@ -60,6 +60,9 @@ const (
 	// Stopped by Annotation holds the information as to why a workspace has stopped
 	StoppedByAnnotation = "controller.devfile.io/stopped-by"
 
+	// RestartAt Annotation holds the the time (unixnano) of when the webhook pod was restarted
+	RestartAtAnnotation = "controller.devfile.io/restart-at"
```

Comment:
Maybe it makes sense to use `restarted-at`, since this annotation will always refer to a time in the past

Reply:
Good idea! Changed to past tense!

Reply:
Kubectl uses `kubectl.kubernetes.io/restartedAt`

Reply:
Ohhhh okay, maybe it makes sense to change it to that then

Reply:
Sorry, as usually I was not clear enough.

I wanted to show that kubectl uses restarted (not restart)

But I did not catch that there is still difference with camel-case and - as word divider.

So, I don't have a strong opinion if one of both is better, I just care that we're consistent on controller side.
Which does not seem to be true even now:
- `workspace_name`
- `discoverable-server`
But it's out of the current PR scope.

We may change them when do workspace -> devworkspace renaming.

Reply:
Ohhh okay, got it :-) makes sense, thanks @sleshchenko